build.gradle

```diff
@@ -107,6 +107,19 @@ subprojects {
         apply from: "${rootProject.projectDir}/${appServicesSrcDir}/build-scripts/substitute-local-appservices.gradle"
     }
 
+    // Define a re-usable task for updating the version in manifest.json (for modules that package a web extension)
```

Comment:
It would be nice to describe why we actually need to do this. You don't mention the reason for this when registering the task, either.

Reply:
Good point. Added docs why we're doing this.